core/devmode/src/main/java/io/quarkus/dev/DevModeContext.java

```diff
@@ -120,6 +123,14 @@ public void setTargetJvmVersion(String targetJvmVersion) {
         this.targetJvmVersion = targetJvmVersion;
     }
 
+    public List<String> getCompilerPluginArtifacts() {
+        return compilerPluginArtifacts;
+    }
+
+    public List<String> getCompilerPluginsOptions() {
+        return compilerPluginsOptions;
+    }
```

Comment:
I would prefer having setters for those two. Similar to `setCompilerOptions`.

Reply:
OK, I'll update

Reply:
Fixed